src/App/Pages/Settings/SettingsPage/SettingsPageListItem.cs

```diff
@@ -22,5 +23,41 @@ public class SettingsPageListItem : ISettingsPageListItem
         public Color SubLabelColor => SubLabelTextEnabled ?
             ThemeManager.GetResourceColor("SuccessColor") :
             ThemeManager.GetResourceColor("MutedColor");
+        public string AutomationId
+        {
+            get
+            {
+                if (!UseFrame)
+                {
+                    var idText = new CultureInfo("en-US", false)
+                    .TextInfo
+                    .ToTitleCase(Name)
+                    .Replace(" ", String.Empty)
+                    .Replace("-", String.Empty);
+                    return $"{idText}Cell";
+                }
+                else
+                {
+                    return "EnabledPolicyCell";
+                }
+            }
+        }
+
+        public string AutomationIdSettingStatus
+        {
+            get
+            {
+                if (!UseFrame)
+                {
+                    var idText = new CultureInfo("en-US", false)
+                    .TextInfo
+                    .ToTitleCase(Name)
+                    .Replace(" ", String.Empty)
+                    .Replace("-", String.Empty);
+                    return $"{idText}SettingValue";
+                }
+                return null;
+            }
+        }
```

Comment:
♻️  Apply same thing as in `AutomationId` and I tend to use failure scenarios at first like here:
```c#
if (UseFrame)
{
    return null;
}

// the other logic
```